README.md

```diff
@@ -0,0 +1,196 @@
+- React front-end
+- Redux state management
+- Integrated back-end
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> No live demo yet.
+
+<!-- - [Live Demo Link]() -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+- Clone this repository by writing `git clone https://github.com/businesskaniki/hello-rails-react.git`
+- Run `bundle install` to install the project's dependencies
+- Run  `npm install` to install the node dependencies
+- Add your database username and password and write `rails db:create` to create the database
+- Run  `rails db:migrate` to migrate the tables to the created database
+- Run  `rails db:seed` to populate the tables with the 
+- Run `rails server` to open the local host
+
+### Prerequisites
+
+In order to run this project you need:
+- Install PostgreSQL in your machine
+- install Ruby and Ruby on rails
+- Install your preferred IDE(Visual Studio is highly recommended)
+- Then run: 
+
+```
+ bundle install
+
+```
+```
+  npm install
+```
+
+### Setup
+
+- Clone this repository to your desired folder
+- Navigate to the cloned repository and follow the aforementioned steps to get started
+
+### Install
+
+Install this project with:
+
+- PostgreSQL
+- Ruby
+- Ruby on Rails
+- Webpack
+- React
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+rspec <relative path to the test file>
+```
+
+### Deployment
+
+- The project has no active deployments yet
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤nichoolas maina
+
+- GitHub: [nick](https://github.com/bussinesskaniki)
+- LinkedIn: [nick](https://twitter.com/businesskaaniki)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+<!-- ## 🔭 Future Features <a name="future-features"></a>
+
+> This project will eventually include:
+
+- [ ] A desktop version
+- [ ] A feature to enable users upload their profile pictures
+- [ ] QR code integration to process payments
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<!-- > I would like to thank all those whose ideas were used in this project. -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- Please ensure you have a valid MIT LICENSE link in your README. The current link needs to be fixed. 😢